aip/aog/3010.md

```diff
@@ -0,0 +1,60 @@
+---
+aip:
+  id: 3010
+  state: draft
+  created: 2020-01-14
+  updated: 2020-01-14
+  type: guidance
+  scope: aog
+js:
+  - /assets/js/graphviz/viz.js
+  - /assets/js/graphviz/lite.render.js
+  - /assets/js/aip/aip-graphviz.js
+permalink: /aog/3010
+---
+
+# Actions on Google Vertical Integration Webhook Format
+
+If an action is part of a vertical program that requires a webhook for Google
+to integrate with the action, the webhook format **must** follow this AIP.
+
+## Guidance
+
+### Custom method name
+
+Custom methods in a vertical program are defined by Google and implemented by
+third-party actions. Guidelines related to names of custom methods in
+[AIP-136](../0136.md) **should** be applied.
+
+- The name of the method **should** be a verb followed by a noun.
+- The name must not contain prepositions (“for”, “with”, etc.).
```

Comment:
** must not **

Reply:
done

Reply:
I see `_must not_` rather than `**must not**`.  Could you update to `**must not**`?

Reply:
ah I must have used only one * and Prettier automatically convert it to _ . fixed now